The "Test via Docker container" fails with errors like:

```
Downloading https://downloads.openwrt.org/releases/24.10-SNAPSHOT/targets/x86/64/packages/Packages.gz
Updated list of available packages in /var/opkg-lists/openwrt_core
Downloading https://downloads.openwrt.org/releases/24.10-SNAPSHOT/packages/x86_64/base/Packages.gz
Updated list of available packages in /var/opkg-lists/openwrt_base
Downloading https://downloads.openwrt.org/releases/24.10-SNAPSHOT/targets/x86/64/kmods/6.6.119-1-484466e2719a743506c36b4bb2103582/Packages.gz
*** Failed to download the package list from https://downloads.openwrt.org/releases/24.10-SNAPSHOT/targets/x86/64/kmods/6.6.119-1-484466e2719a743506c36b4bb2103582/Packages.gz

Downloading https://downloads.openwrt.org/releases/24.10-SNAPSHOT/packages/x86_64/luci/Packages.gz
Updated list of available packages in /var/opkg-lists/openwrt_luci
Downloading https://downloads.openwrt.org/releases/24.10-SNAPSHOT/packages/x86_64/packages/Packages.gz
Updated list of available packages in /var/opkg-lists/openwrt_packages
Downloading https://downloads.openwrt.org/releases/24.10-SNAPSHOT/packages/x86_64/routing/Packages.gz
Collected errors:
Updated list of available packages in /var/opkg-lists/openwrt_routing
 * opkg_download: Failed to download https://downloads.openwrt.org/releases/24.10-SNAPSHOT/targets/x86/64/kmods/6.6.119-1-484466e2719a743506c36b4bb2103582/Packages.gz, wget returned 8.
Downloading https://downloads.openwrt.org/releases/24.10-SNAPSHOT/packages/x86_64/telephony/Packages.gz
Updated list of available packages in /var/opkg-lists/openwrt_telephony
Downloading file:///ci/Packages.gz
Updated list of available packages in /var/opkg-lists/packages_ci
Error: Process completed with exit code 1.
```

Seems to be unrelated.

Can someone restart the build please? Maybe it was a transient error?

--- a/admin/openwisp-config/Makefile
+++ b/admin/openwisp-config/Makefile
@@ -5,14 +5,14 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=openwisp-config
-PKG_VERSION:=1.2.0
+PKG_VERSION:=1.2.1
 PKG_RELEASE:=1
 
 PKG_MAINTAINER:=Federico Capoano <f.capoano@openwisp.io>
 PKG_LICENSE:=GPL-3.0-or-later
 
 PKG_SOURCE_URL:=https://github.com/openwisp/openwisp-config.git
-PKG_MIRROR_HASH:=30258c3ef4895fbf6e4fed8caee9d0dfbf05aebebd52604d75febac1a11d78bd
+PKG_MIRROR_HASH:=f709f3aaf2e9ad61b2df378b7e6cfae7050d5f1d8c4c4bc038e92809c843c4b7
 PKG_SOURCE_PROTO:=git
 PKG_SOURCE_VERSION:=$(PKG_VERSION)
 

